src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -968,3 +971,60 @@
 type: command
 short-summary: Update metadata for an IoT Central application.
 """
+
+helps['iot hub private-endpoint-connection approve'] = """
+type: command
+short-summary: Approve a private endpoint connection to an IoT Hub.
+examples:
+  - name: Approve a private endpoint connection to an IoT Hub by name
+    text: >
+        az iot hub private-endpoint-connection approve -n MyPrivateEndpointConnection --hub-name MyIotHub -g MyGroup --desc 'Approved connection'
+  - name: Approve a private endpoint connection to an IoT Hub by ID
+    text: >
+        az iot hub private-endpoint-connection approve --id {id} --desc 'Approved connection'
+"""
+
+helps['iot hub private-endpoint-connection reject'] = """
+type: command
+short-summary: Reject a private endpoint connection to an IoT Hub.
+examples:
+  - name: Reject a private endpoint connection to an IoT Hub by name
+    text: >
+        az iot hub private-endpoint-connection reject -n MyPrivateEndpointConnection --hub-name MyIotHub -g MyGroup --desc 'Rejected connection'
+  - name: Reject a private endpoint connection to an IoT Hub by ID
+    text: >
+        az iot hub private-endpoint-connection reject --id {id} --desc 'Rejected connection'
+"""
+
+helps['iot hub private-endpoint-connection show'] = """
+type: command
+short-summary: Show details about a private endpoint connection to an IoT Hub.
+examples:
+  - name: Show details about a private endpoint connection to an IoT Hub by name
+    text: >
+        az iot hub private-endpoint-connection show -n MyPrivateEndpointConnection --hub-name MyIotHub -g MyGroup
+  - name: Show details about a private endpoint connection to an IoT Hub by ID
+    text: >
+        az iot hub private-endpoint-connection show --id {id}'
```

Comment:
I realize `--id` is used in spec, but in reality `--id` is not used anywhere in the `IoT extension` and it feels weird/non-cohesive that `--id` is recommended here. It is far easier to provide the IoT Hub name (which is globally unique) versus having to put in my ARM resource Id value.

Reply:
This comment isn't necessarily for you to do anything but more awareness.